tools/cuds-prioritized-attribution/test/tests/load_all_test_data.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
run shellcheck and use apache header on this and all shell scripts.

Reply:
Done